README.md

```diff
@@ -87,6 +95,7 @@ memory corruption bugs.
         colcon-mixin-name: asan
         colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/3e627e0fa30db85aea05a50e2c61a9832664d236/index.yaml
         package-name: my_package
+        target_ros2_distro: foxy
```

Comment:
Note that https://github.com/ros-tooling/action-ros-ci/pull/278/ uses hyphens in the names `target-ros1-distro` and `target-ros2-distro`, not underscores.

Reply:
Nice catch!

Reply:
Yeah good catch, thanks for that